build/blueprints/component/files/src/patternfly/components/__moduleName__/styles.scss

```diff
@@ -6,26 +6,25 @@
 // The value of component scoped variables is always defined by a global variable:
 :root {
   // Example block variable
-  --{{bemName}}--Color: var(--pf-global--dark--Color);
+  --{{bemName}}--Color: var(--pf-global--Color--dark-100);
```

Comment:
we will have to make a note to go back and change this again when @andresgalante updates these variables. This will be come `--pf-global--Color--100`

Reply:
@matthewcarleton in case it helps, right clicking on a dir in the VSCode explorer and using "find in folder" will surface any instance of that string in case it needs to be updated anywhere else.

You can also `grep` the repo.

```
cmichael:patternfly-next cmichael$ grep -ri '\-\-pf\-global\-\-Color\-\-dark\-100' *
build/blueprints/component/files/src/patternfly/components/__moduleName__/styles.scss:  --{{bemName}}--Color: var(--pf-global--Color--dark-100);
src/patternfly/utilities/variables.scss:  --pf-global--Color--dark-100: $pf-global--Color--dark-100;
src/patternfly/components/Dropdown/styles.scss:  --pf-c-dropdown__toggle--Color: var(--pf-global--Color--dark-100);
src/patternfly/components/Dropdown/styles.scss:  --pf-c-dropdown__menu-item--Color: var(--pf-global--Color--dark-100);
```